scala/revenj-core/src/main/scala/net/revenj/database/postgres/converters/NumberConverter.scala

```diff
@@ -101,6 +101,43 @@ object NumberConverter {
     value
   }
 
+  def serialize(value: Short, buf: Array[Char]): Int = {
```

Comment:
I left this out on purpose. This is kind of useless, that why it redirected to int conversion